assembler/style.css

```diff
@@ -14,27 +14,18 @@ Text Domain: assembler
 Tags: blog, one-column, three-columns, wide-blocks, block-patterns, custom-colors, custom-logo, custom-menu, editor-style, featured-images, full-site-editing, rtl-language-support, style-variations, template-editing, theme-options, threaded-comments, translation-ready
 */
 
-/* Progresive enhancement to reduce widows and orphans. */
+/* Make link colors match text colors. */
 
-h1,
-h2,
-h3,
-h4,
-h5,
-h6,
-blockquote {
-	text-wrap: balance;
+h1 a, h2 a, h3 a, h4 a, h5 a, h6 a {
+    color: inherit !important;
```

Comment:
Hey @richtabor 👋🏼🙂 Looks like the `!important` here caused a small regression here: https://github.com/Automattic/themes/issues/8193.

I wonder what was the underlying need for this color override. As when removing it in my tests, it doesn't seem to cause any issues. 🤔 I suspect there might be some case I am missing.

Reply:
Here's a PR with the fix ready for review: https://github.com/Automattic/themes/pull/8206.

Reply:
I'll take a look, thanks :)